app/renderer/components/Session/NewSessionForm.js

```diff
@@ -3,9 +3,20 @@ import { Button, Input, Modal, Form, Row, Col, Select } from 'antd';
 import FormattedCaps from './FormattedCaps';
 import CapabilityControl from './CapabilityControl';
 import SessionStyles from './Session.css';
+import {remote} from 'electron';
+import { HEIGHT_OF_SERVICE_CONFIG_AREA } from './Session.js';
+
 const {Item: FormItem} = Form;
 const {Option} = Select;
 
+const styles = {
+  // Got warning messages not to set by CSS
```

Comment:
I'd also use CSS to define styles. I'm surprised that electron prints such warning

Reply:
I found a wrong syntax in my previous CSS implementation